app/javascript/components/ansible-playbook-edit-catalog-form/helper.js

```diff
@@ -0,0 +1,315 @@
+import { useFieldApi, useFormApi } from '@@ddf';
+import React from 'react';
+import { Button, TextInput, Checkbox, Select, SelectItem } from 'carbon-components-react';
```

Comment:
We can remove the Select and SelectItem since they are unused

Reply:
made the changes.Thanks